_templates/api-story-card/with-prompt/src/components/App/index-card.ejs.t

```diff
@@ -0,0 +1,9 @@
+---
+to: packages/<%=package%>/src/components/App/index.js
+inject: true
+before: '</PageLayout>'
+skip_if: <<%=StoryCardName%>
```

Comment:
I'm not 100% or even 40% familiar with hygen, but I feel like `<<%=StoryCardName%>` isn't the syntax that should be here. Should the opening carat be removed or a final `/>` be added?

Reply:
Ahh. Ok. That is it. I see in the addition below that if `<ComponentName` isn't present then we're adding the section below.